docs/UserGuide.md

```diff
@@ -320,6 +317,28 @@ Examples:
 * `list` followed by `share 2` copies the information of the second contact in the list displayed by SociaLite.
 * Clicking on the "Share" button of Annie Baker's name card copies her contact information.
 
+<br>
+
+### Pinning and unpinning a contact card: `pin`/`unpin`
+
+Pins/Unpins the contact card of the specified person to the system clipboard. The pinned contact cards will be alphabetically ordered at the top of the person list.
```

Comment:
I think you mean "Pins/Unpins the contact card of the specified person at the top of the person list" instead of "system clipboard"? Maybe we can change this to:

`pin` affixes the specified contact card onto the top of the person list. 
`unpin` removes the specified contact card from the top of the person list, provided it was previously pinned.
The pinned contact cards will be alphabetically ordered.